requirements/datatype_image_extras.txt

```diff
@@ -2,6 +2,7 @@ matplotlib
 fiftyone
 classy_vision
 vissl>=0.1.5
+git+https://github.com/learnables/learn2learn.git
```

Comment:
Use (the now published) official version of the library

Reply:
I need my PR to be merged in their repo first.